test/unit/api_spec.js

```diff
@@ -341,13 +341,32 @@ describe('api', function() {
       });
     });
     it('gets non-existent page', function(done) {
-      var promise = doc.getPage(100);
-      promise.then(function () {
-        done.fail('shall fail for non-existent page');
-      }, function(data) {
-        expect(data instanceof Error).toEqual(true);
-        done();
+      var outOfRangePromise = doc.getPage(100);
+      var nonIntegerPromise = doc.getPage(2.5);
+      var nonNumberPromise = doc.getPage('pageNumber');
```

Comment:
Whoops, this should have been a number given as a string (e.g. `var nonNumberPromise = doc.getPage('pageNumber');`), I've pushed a new version that fixes this.